Python/Product/Analysis/LanguageServer/Server.cs

```diff
@@ -301,6 +301,12 @@ private sealed class ReloadModulesQueueItem : IAnalyzable {
         public void SetSearchPaths(IEnumerable<string> searchPaths) => _analyzer.SetSearchPaths(searchPaths.MaybeEnumerate());
         public void SetTypeStubSearchPaths(IEnumerable<string> typeStubSearchPaths) => _analyzer.SetTypeStubPaths(typeStubSearchPaths.MaybeEnumerate());
 
+        public event EventHandler<FileFoundEventArgs> OnFileFound;
```

Comment:
Do we need this for tests? I'm not seeing anywhere it's essential (and I can't get the `ChildFileAnalyzed` event to trigger in master right now either, which probably means we don't care about it).

Reply:
It looks functional. `PythonAnalyzer` maintains `_projectFilesByUri`